src/Component.php

```diff
@@ -5,12 +5,155 @@
 namespace MyComponent;
 
 use Keboola\Component\BaseComponent;
+use Keboola\Component\UserException;
+use Keboola\Csv\CsvFile;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+use Symfony\Component\Serializer\Encoder\JsonDecode;
+use Symfony\Component\Serializer\Encoder\JsonEncode;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
 
 class Component extends BaseComponent
 {
+    /**
+     * @var JsonEncode
+     */
+    private $jsonEncode;
+
+    /**
+     * @var JsonDecode
+     */
+    private $jsonDecode;
+
+    /**
+     * @var Filesystem
+     */
+    private $fs;
+
+    private function validateManifest(array $manifest, string $fileName) : void
```

Comment:
~Bývá zvykem mít~ Mám radši, když jsou metody od nejobecnější k nejkonkrétnějším - tj. nejdřív si přečtu `run()` a pak jak scrolluju, potkávám další metody. (PhpStorm to [umí sám od sebe](https://blog.jetbrains.com/phpstorm/2013/03/arrange-your-code-automatically-with-phpstorm-rearranger/))

Reply:
budiz :)